### ISSUE TITLE
Applying New datasets

### ISSUE THREAD

**Issue by Jay0412** [thunlp/PL-Marker#28]
Hello again, and now I want to apply new datasets which look like the below.
![image](https://user-images.githubusercontent.com/43334793/171677705-f7dfe176-d267-45eb-af9f-56e3275c36d1.png)
This dataset has a few different categories but, I think it doesn't matter as long as they have the same format(SciERC) for essential categories(setences, ner, relations).
So I use this dataset on ner training but, I get very poor result of it.. (even train loss is very small)
![image](https://user-images.githubusercontent.com/43334793/171688667-11315306-178b-4bb3-9510-a1bd7122daa8.png)
![image](https://user-images.githubusercontent.com/43334793/171688723-b83727de-5f2f-4314-939b-a20a3a70a7b9.png)
Of course, I make new data label and seed in run_acener.py
I try to find what's the reason of the problem but, I still couldn't find it.
Did I do anything wrong when I applied new data or is it just overfitting?

**Comment by YeDeming** [comment]
The loss is very low now. What's the results when you test on the training set?

**Comment by Jay0412** [comment]
This is the result that I got from the test on the dev.
![image](https://user-images.githubusercontent.com/43334793/171815918-9695ab6f-a1b7-4d91-bb08-8efd0be3cb7d.png)
This is the result that I got from the test on the train.
![image](https://user-images.githubusercontent.com/43334793/171816035-eb22067e-3b19-4631-a0be-3a2caf762148.png)
I have no idea but, testing on the dev and test got the same results..

**Comment by YeDeming** [comment]
The training loss is very low means that the model can get a high performance on the training set, but it didn't. Maybe you should check this.  
Sorry, I am busy for the EMNLP submission now and have no time to check this dataset in details.
